ee/codegen/src/__test__/__snapshots__/generate-code.test.ts.snap

```diff
@@ -123,18 +123,18 @@ class InnerMapNode(MapNode):
 exports[`generateCode > should generate code for %1 scheduled-trigger.ts > triggers/scheduled.py 1`] = `
 "from datetime import datetime
 
-from vellum.workflows.triggers import BaseTrigger
+from vellum.workflows.triggers import ScheduleTrigger as TriggersScheduleTrigger
 
 
-class ScheduleTrigger(BaseTrigger):
+class ScheduleTrigger(TriggersScheduleTrigger):
     current_run_at: datetime
     next_run_at: datetime
```

Comment:
[oos] we should be able to now avoid generating these since it's redundant with the base class